frontend/src/lib/components/icons.tsx

```diff
@@ -264,13 +264,13 @@ export function IconKey(props: LemonIconProps): JSX.Element {
     )
 }
 
-/** Material Design Terminal icon. */
-export function IconTerminal({ style }: { style?: CSSProperties }): JSX.Element {
+/** Material Design Console Line icon. */
+export function IconConsoleLine({ width = 24, height = 24, ...props }: React.SVGProps<SVGSVGElement>): JSX.Element {
```

Comment:
Hard to say for sure as this is more "taste" than anything else, but I personally think this icon looks worse... 
<img width="684" alt="Screenshot 2023-01-17 at 18 19 28" src="https://user-images.githubusercontent.com/2536520/212967622-a2e9453d-26ce-4922-8e4f-0028fce8e9d9.png">

The thickness of the icon really stands out. The argument that it is similar to the window icon makes sense, but I don't think it is so bad, especially with this icon as the replacement 😅

Reply:
Hm maybe we can ask @lottiecoxon or @corywatilo for some advice here?

Reply:
I reverted this and also went in and resized some of the icons in Figma and now it seems to look much nicer. I think we should opt for changing the Window icon rather than changing the console logs icon. It doesn't render nicely often so I think we could revisit that properly later